Common/src/Pollster/CommonDataRequestProcessor.cs

```diff
@@ -0,0 +1,81 @@
+﻿// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2022. All rights reserved.
+//   W. Kirschenmann   <wkirschenmann@aneo.fr>
+//   J. Gurhem         <jgurhem@aneo.fr>
+//   D. Dubuc          <ddubuc@aneo.fr>
+//   L. Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F. Lemaitre       <flemaitre@aneo.fr>
+//   S. Djebbar        <sdjebbar@aneo.fr>
+//   J. Fonseca        <jfonseca@aneo.fr>
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+using ArmoniK.Api.gRPC.V1;
+using ArmoniK.Core.Common.Storage;
+using ArmoniK.Core.Common.Utils;
+
+using Microsoft.Extensions.Logging;
+
+namespace ArmoniK.Core.Common.Pollster;
+
+public class CommonDataRequestProcessor : IProcessReplyProcessor
+{
+  private readonly IObjectStorage                           resourcesStorage_;
+  private readonly IAsyncPipe<ProcessReply, ProcessRequest> pipe_;
+  private readonly ILogger                                  logger_;
+
+  public CommonDataRequestProcessor(IObjectStorage                           resourcesStorage,
+                       IAsyncPipe<ProcessReply, ProcessRequest> pipe,
+                       ILogger                                  logger)
+  {
+    resourcesStorage_ = resourcesStorage;
+    pipe_             = pipe;
+    logger_           = logger;
+  }
+
+  public async Task AddProcessReply(ProcessReply      processReply,
+                                    CancellationToken cancellationToken)
+  {
+    await pipe_.WriteAsync(new ProcessRequest
+                           {
+                             CommonData = new ProcessRequest.Types.DataReply
+                                          {
+                                            ReplyId = processReply.RequestId,
+                                            Init = new ProcessRequest.Types.DataReply.Types.Init
+                                                   {
+                                                     Key   = processReply.CommonData.Key,
+                                                     Error = "Common data are not supported yet",
+                                                   },
+                                          },
+                           })
+               .ConfigureAwait(false);
+  }
+
+  public bool IsComplete()
+    => true;
```

Comment:
When you are asking if the task is complete, the answer is always True?

Reply:
In current implementation, yes.

Reply:
IsComplete is not used so maybe we need to remove it ? @wkirschenmann

Reply:
You're right !!!!! 
we need to store the task used for the reply and test for its completion.